submissions/RubyAdeniregun/HTML/Product Page.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>DUNA STORES</title>
+</head>
+<body>
+   <hi><b>DUNA STORES</b></hi> 
```

Comment:
<hi> is not an html tag. I think you wanted to use <h1> instead.

 Kindly correct this.

Reply:
Yes, you are right, I meant to use the h1 tag. I have corrected it. As for the "b" tag, I wanted to use it to bolden the word "Duna Stores". Just like the strong tag. I thought the "b" tag and the strong tag do the same thing?